blockchain/action/execution_test.go

```diff
@@ -72,3 +72,32 @@ func TestExecutionSerializeDeserialize(t *testing.T) {
 	require.Equal(ex.Hash(), newex.Hash())
 	require.Equal(ex.TotalSize(), newex.TotalSize())
 }
+
+func TestExecutionToJSONFromJSON(t *testing.T) {
+	require := require.New(t)
+	executor, err := iotxaddress.NewAddress(true, chainid)
+	require.Nil(err)
```

Comment:
Prefer to use`require.NoError`

Same for the error assertions bellow

Reply:
`require.Nil` is used in other test functions writed by others, so I use it here. Should I change all of them?